velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -748,6 +764,17 @@ bool ParquetRowReader::isRowGroupBuffered(int32_t rowGroupIndex) const {
   return readerBase_->isRowGroupBuffered(rowGroupIndex);
 }
 
+std::unique_ptr<BloomFilter> ParquetRowReader::getBloomFilter(
+    const uint32_t rowGroupId,
+    const uint32_t column) {
+  return facebook::velox::parquet::getBloomFilter(
```

Comment:
This looks a bit hacky to me. It will read and deserialize the bloom filter every time it's called for the same rowGroupId and column, though the callers are not doing this so far. When a user sees an interface like this, he or she may naturally think the  ParquetData::getBloomFilter() would not be doing the work and should have a single entry point, which is when you filter the row groups. But the callsite here is reading and deserializing the bloom filter again. I understand this is just for tests, but the existence of this function here would not suggest that, but instead, it looks like some real business logic code. If we make the get the bloom filter as a member of ParquetData, here we can just get the bloom filter by getting its the column'th child SelectiveColumnReader and the ParquetData it owns, then we can get the Bloom filter.